profile/README.md

```diff
@@ -55,13 +61,17 @@ The team should explore various potential solutions and figure out trade-offs wi
 
 Specifically for nixpkgs, this includes all the past history, with all its approaches, refactorings, complaints and testimonies.
 
-### Reaching consensus
+### Attempt at consensus
 
-To decide on a solution, consensus has to be reached among _all_ team members. If consensus cannot be reached yet it indicates that there is some fundamental misundestanding or difference in values between team members. This needs to be reconciled by:
+To decide on a solution, ideally consensus should to be reached among the team members. If consensus cannot be reached yet it can mean that there is a fundamental misundestanding or difference in values between team members. This should be reconciled by:
 - Going back to solution exploration to find new alternatives or gather more information or support for arguments
 - Trying to better understand each other by talking and listening
 - Being willing to compromise
 
+### Last resort: Voting
+
+As a last resort, if still no consensus can be reached after everything has been tried, [@infinisil:matrix.org](https://matrix.to/#/@infinisil:matrix.org) can decide to hold a vote to decide on the issue. The members that don't win in the vote should be accomodated to the best extent possible.
```

Comment:
This is motivated by http://new-compass.net/articles/against-consensus-dissensus mentioned during the meeting. For now I put myself as the facilitator for votes, but I hope to never need this